recipe/gen_patch_json.py

```diff
@@ -994,6 +994,11 @@ def _gen_new_index(repodata, subdir):
             new_depends = record.get("depends", [])
             new_depends.append("libgcc-ng <=9.3.0")
             record["depends"] = new_depends
+        
+        if record.get("timestamp", 0) < 1640101398654:  # 2021-12-21
```

Comment:
We should restrict this to pandas. We don't want to add a setuptools dependency to *all* packages.
```suggestion
        if record_name == "pandas" and record.get("timestamp", 0) < 1640101398654:  # 2021-12-21
```

Reply:
Oh I don't think this only applies to pandas though. This applies to every package that ever used `LooseVersion` from `distutils` without pinning setuptools. Right?

Reply:
But maybe pandas is a good enough place to start I suppose.

Reply:
If we can put together a census of packages, we can add them here later. 

Also, @CJ-Wright may have some tools to help us examine which packages with which versions use this symbol.

Reply:
Ah ok yeah I was going to start tracing back which versions different packages switch to packaging. I don't think it's too terrible to do this based on timestamp though if that ends up being hard. 

Ping @@jorisvandenbossche to make sure pandas is ok with this patch.